platform/security/core/security-core-impl/src/main/java/ddf/security/config/impl/ConfigurationSecurityLogger.java

```diff
@@ -56,8 +56,10 @@ public void configurationEvent(ConfigurationEvent event) {
         // there was no subject change was caused by an update to the config file on the filesystem
         SecurityLogger.auditWarn("Configuration {} via filesystem: {}", type, updatedConfiguration);
       }
+    } catch (VirtualMachineError vme) {
+      throw vme;
     } catch (Throwable e) {
-      LOGGER.error("Error auditing config update for {}", event.getPid(), e);
+      LOGGER.error("Error auditing config update for " + event.getPid(), e);
```

Comment:
❓ why was this change made, the log format seemed correct for slf4j?

Reply:
because it wasn't logging the error. The prototype for the error() method that accepts a throwable doesn't accept a format message and parameters